src/scripts/GameConstants.d.ts

```diff
@@ -70,6 +70,7 @@ namespace GameConstants {
     declare const FREEZE_MULCH_MULTIPLIER: number;
     declare const WANDER_RATE: number;
     declare const BerryColor: string[];
+    declare const BerryFirmness: string[];
```

Comment:
BerryFirmness doesn't exist in game constants, the BerryColor array I think you are imitating is actually the array of hex color codes, not the enum. No need for a manual declaration, your /enums/BerryFirmness.ts will generate one.